src/store/getters.js

```diff
@@ -57,7 +58,7 @@ export default {
       const account = accountId ? getters.accountItem(accountId) : null
       const _accountType = account?.type || accountType
       const _accountIndex = account?.index || accountIndex
-      const { chain } = getters.cryptoassets[asset]
+      const { chain } = getters.cryptoassets[asset] || cryptoassets[asset]
```

Comment:
Why is that required?

Reply:
const { chain } = getters.cryptoassets[asset]  - takes all the assets that we have currenlty in the walleet cache, but if we add new assets in @liquality/cryptoassets package - they are not going to be detected.  

That's why the change is required

Reply:
I've removed this and moved it to another PR